static/sass/custom/_modal.scss

```diff
@@ -0,0 +1,36 @@
+.p-modal {
```

Comment:
Should these be contributed back up to vanilla? Add the fade-in-out functionality to the upstream...